### PR TITLE
feat: Role Invite: Allow users to resend invitations of event transfer from the settings page itself.

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes https://github.com/fossasia/open-event-server/issues/7953

#### Short description of what this resolves:
self explantory

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/events/view/settings/options.js [hunk 1]
```diff
@@ -67,6 +67,25 @@ export default class OptionsController extends Controller {
...
+      const invite =  this.model.roleInvites.filter(invite => invite.email === this.currentInvite.email)[0];
+      if (invite.id) {
+        const res = await this.loader.post('/role-invites/' + invite.id + '/resend-invite');
+        if (res.success) {
+          this.notify.success(this.l10n.t('Invite resent successfully'),
+            {
+              id: 'resend_invite_succ'
+            });
+        } else {
+          this.notify.error(this.l10n.t('Oops something went wrong. Please try again'));
```
Comment (maze-runnar) [inline app/controllers/events/view/settings/options.js:79]: errorhandler

File: app/controllers/events/view/settings/options.js [hunk 2]
```diff
@@ -67,6 +67,25 @@ export default class OptionsController extends Controller {
...
+        }
+        this.setProperties({
+          'isConfirmEventTransferModalOpen' : false,
+          'checked'                         : false
+        });
+        this.set('isLoading', false);
```
Comment (maze-runnar) [inline app/controllers/events/view/settings/options.js:85]: what need to set loading false here. it is already on line 100

Comment (ranjitkshah) [reply app/controllers/events/view/settings/options.js:85]: due to return statement that set loading will not work

**Review by sachinchauhan2889 — APPROVED** [review]

**Review by maze-runnar — APPROVED** [review]

**Review by MeghalBisht — APPROVED** [review]
Works! 🥂

**Pull request merged**